slidingwindow.h

```diff
@@ -1768,19 +1768,19 @@ template<unsigned int ConvKernelDim_x,
 		 unsigned int Input_precision,
 		 unsigned int IFMDim_x,
 		 unsigned int OFMDim_x,
-		 unsigned int SIMD,
 		 unsigned int Stride_x,
          unsigned int Dilation_x,
+		 unsigned int SIMD,
 		 typename R>
-void ConvolutionInputGenerator_1D_dilated_dws(
+void ConvolutionInputGenerator_1D_dws_naive(
 		stream<ap_uint<SIMD*Input_precision> > & in,
 		stream<ap_uint<SIMD*Input_precision> > & out,
 		const unsigned int numReps,
 		R const &r) {
   CASSERT_DATAFLOW(IFMChannels % SIMD == 0);
 
-  const unsigned int multiplying_factor = IFMChannels/SIMD;
-  const unsigned int cycles_write_block = (OFMDim_x * ConvKernelDim_x * multiplying_factor);
+  const unsigned int multiplying_factor = IFMChannels / SIMD;
+  const unsigned int cycles_write_block = OFMDim_x * ConvKernelDim_x * multiplying_factor;
```

Comment:
Not a hard requirement, just a suggestion:
When you are touching such code anyhow, you might as well make these trivial compile-time constants proper C++ `constexpr`s.

Reply:
Changed it.